src/com/gfredericks/test/chuck/clojure_test.cljc

```diff
@@ -1,23 +1,36 @@
 (ns com.gfredericks.test.chuck.clojure-test
   (:require [clojure.test.check :as tc]
+            [clojure.test.check.clojure-test :as tcct]
```

Comment:
`tcct` require is not needed anymore, now that `with-test-out*` had to be brought here, right?